packages/amplify-e2e-core/src/utils/index.ts

```diff
@@ -33,58 +34,95 @@ export function deleteAmplifyDir(root: string) {
   rimraf.sync(path.join(root, 'amplify'));
 }
 
-export function overrideFunctionSrc(root: string, name: string, code: string) {
-  let indexPath = path.join(getPathToFunction(root, name), 'src', 'index.js');
-  fs.writeFileSync(indexPath, code);
+export function loadFunctionTestFile(fileName: string) {
+  const functionPath = getTestFileNamePath(fileName);
+  return fs.readFileSync(functionPath, 'utf-8').toString();
 }
 
-export function overrideFunctionCode(root: string, name: string, fileName: string) {
-  let indexPath = path.join(getPathToFunction(root, name), 'src', 'index.js');
-  let functionPath = path.join(__dirname, '..', '..', '..', 'amplify-e2e-tests', 'functions', fileName);
-  fs.copySync(functionPath, indexPath);
+export function addNodeJSDependencies(root: string, functionName: string, dependencies: string[]) {
```

Comment:
nit: function name can just be `addNodeDependencies` to match naming convention used below

Reply:
I just moved it, but will rename :)